dataikuapi/dss/ml.py

```diff
@@ -3423,3 +3423,14 @@ def guess(self, prediction_type=None, reguess_level=None):
             "PUT",
             "/projects/%s/models/lab/%s/%s/guess" % (self.project_key, self.analysis_id, self.mltask_id),
             params = obj)
+
+
+class DSSMLTaskQueues(object):
+    """
+    Object containing a list of MLTask queues
+    """
+    def __init__(self, queues):
+        self.queues = queues
```

Comment:
Typically we now build these kind of result objects with the whole dict as argument, and make that dict available via `.get_raw()` (with rtype dict). And for example, if we want to expose `.queues`, that could be a `@property` method that returns the internal dict's `["queues"]` item.